### ISSUE TITLE
Augmented Reality (AR) View

### ISSUE THREAD

**Issue by sv410** [Vishwajith-Shettigar/NextGen#93]
Integrate an AR view into the app that allows users to see the real-time locations of nearby people overlaid on their device’s camera view. This feature enhances the experience by making it more immersive and interactive.
Key Functionalities
AR Overlay:
Display real-time locations of nearby users on the camera view using AR markers.
Each marker shows the user’s name and distance.
Interaction with Markers:
Tapping on an AR marker can open options to chat, start a video call, or view the user’s profile.
AR Navigation:
Provide AR-based navigation to guide users to the exact location of their nearby friends.
Technologies Used
ARCore (for Android): To handle the AR functionalities.
Firebase: For real-time data synchronization.
Google Maps API: To get the coordinates of nearby users.

**Comment by Vishwajith-Shettigar** [comment]
Are you sure google ARcore could handle our requirements with minimal efforts ?

**Comment by sv410** [comment]
@Vishwajith-Shettigar yes i can try on this pls level 3 and add GSSOC , hacktoberfest if u like my work

**Comment by sv410** [comment]
@Vishwajith-Shettigar i have done creating ARActivity.kt: Main Kotlin file handling AR logic.
activity_ar.xml: Layout for AR activity
ar_marker_layout.xml: Layout for AR markers
With this setup, you can display real-time locations of nearby users overlaid on their camera view.

**Comment by Vishwajith-Shettigar** [comment]
@sv410  create draft PR.

**Comment by Vishwajith-Shettigar** [comment]
> @sv410 create draft PR.

Give me link, because I can't see here.
